engine/runtime/src/main/java/org/enso/interpreter/runtime/PatchedModuleValues.java

```diff
@@ -91,7 +91,7 @@ boolean simpleUpdate(SimpleUpdate update) {
     }
     if (collect.isEmpty()) {
       // only search for new literals when none have been found
-      updateFunctionsMap(update, methods.values(), collect);
+      updateFunctionsMap(update, methods, collect);
```

Comment:
This place uses `method.values()` just to obtain list of all `Function` and their `RootNode` instances. E.g. possibly `getAllMethods()` could return just `Collection<Function>`.